test/integration/host_js_test.rb

```diff
@@ -315,6 +332,23 @@ class HostJSTest < IntegrationTestWithJavascript
       click_link 'Parameters'
       assert_equal find("#s2id_host_lookup_values_attributes_#{lookup_key.id}_value .select2-chosen").text, "true"
     end
+
+    test 'changing host group updates parameters tab' do
```

Comment:
They're in different contexts, but I've changed this one to "setting host group [..]" as it's not changing from one group to another.